lib/formatters.js

```diff
@@ -134,14 +134,32 @@ function colorize(color, str) {
  * @deprecated
  */
 function formatTrace(level, context, message, args, err) {
-  args.unshift(
-      templateTrace + message,
-      (new Date()).toISOString(),
-      level,
-      context.corr || notAvailable,
-      context.trans || notAvailable,
-      context.op || notAvailable
-  );
+
+  var recontext = {
+    time: (new Date()).toISOString(),
+    lvl: level,
+    corr: context.corr || notAvailable,
+    trans: context.trans || notAvailable,
+    op: context.op || notAvailable
```

Comment:
nope, because this is a special case for BW compatibility, to have these fields always in the final trace, even when they are not in the context (and therefore not iterated by `Object.keys()`).